app/javascript/custom-tagged-plate/components/CustomTaggedPlate.spec.js

```diff
@@ -15,7 +15,7 @@ import {
   exampleTag2Group,
   exampleTag2GroupLonger,
   exampleQcableData,
-  exampleTag1GroupChromium
+  exampleTag1GroupChromium,
```

Comment:
The comma on the last item in a list thing, is that the standard with JS? came up in discussion with another developer and we weren't sure if/why it should be there other than a convenience thing if you want to add more items later...

Reply:
It depends on how the linter is configured, as essentially it's optional. It seems most style guides are trending towards including it. I assume ES lint is currently set up not to care either way, and this was probably added by VS Code before I turned of prettier in repos where it isn't explicitly enabled.

But yeah, you're right. The main driver for this is that if you add a new item, the diff will be only one line, rather than showing the new item, and the addition of the comment on the previous line. This also potentially makes merging slightly easier if there is a conflict with two entries added, as you can just accept both changes.

For some reason, the ruby community has settled on the opposite, favouring the lack of a trailing comma. This is despite ruby supporting it just fine:
```
irb(main):002:0> [1,2,3,]
=> [1, 2, 3]
irb(main):003:0> {a:1,b:2,c:3,}
=> {:a=>1, :b=>2, :c=>3}
```